tutorials/asr/09_ASR_for_telephony_speech.ipynb

```diff
@@ -0,0 +1,423 @@
+    "3. Connect to an instance with a GPU (Runtime -> Change runtime type -> select \"GPU\" for hardware accelerator)\n",
+    "4. Run this cell to set up dependencies.\n",
+    "5. Restart the runtime (Runtime -> Restart Runtime) for any upgraded packages to take effect\n",
+    "\"\"\"\n",
+    "# If you're using Google Colab and not running locally, run this cell.\n",
+    "\n",
+    "## Install dependencies\n",
+    "!pip install wget\n",
+    "!apt-get install sox libsndfile1 ffmpeg\n",
+    "!pip install unidecode\n",
+    "!pip install matplotlib>=3.3.2\n",
+    "\n",
+    "## Install NeMo\n",
+    "BRANCH = 'v1.0.0'\n",
+    "!python -m pip install git+https://github.com/NVIDIA/NeMo.git@$BRANCH#egg=nemo_toolkit[all]\n",
+    "\n",
+    "## Grab the config we'll use in this example\n",
+    "!mkdir configs\n",
+    "!wget -P configs/ https://raw.githubusercontent.com/NVIDIA/NeMo/$BRANCH/examples/asr/conf/config.yaml\n",
+    "\n",
+    "\"\"\"\n",
+    "Remember to restart the runtime for the kernel to pick up any upgraded packages (e.g. matplotlib)!\n",
+    "Alternatively, you can uncomment the exit() below to crash and restart the kernel, in the case\n",
+    "that you want to use the \"Run All Cells\" (or similar) option.\n",
+    "\"\"\"\n",
+    "# exit()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "v1Jk9etFlRzf"
+   },
+   "source": [
+    "# Telephony speech (8 kHz)\n",
+    "This notebook covers general recommendations for using NeMo models with 8 kHz speech. All the pretrained models currently available through NeMo are trained with audio at 16 kHz. This means that if the original audio was sampled at a different rate, it's sampling rate was converted to 16 kHz through upsampling or downsampling. One of the common applications for ASR is to recognize telephony speech which typically consists of speech sampled at 8 kHz.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Mixed sample rate\n",
+    "Most of the pretrained English models distributed with NeMo are trained with mixed sample rate data, i.e. the training data typically consists of data sampled at both 8 kHz and 16 kHz. As an example pretrained Citrinet model \"stt_en_citrinet_1024\" was trained with the following datasets. \n",
+    "* Librispeech 960 hours of English speech\n",
+    "* Fisher Corpus\n",
+    "* Switchboard-1 Dataset\n",
+    "* WSJ-0 and WSJ-1\n",
+    "* National Speech Corpus - 1\n",
+    "* Mozilla Common Voice\n",
+    "\n",
+    "Among these, Fisher and Switchboard datasets are conversational telephone speech datasets with audio sampled at 8 kHz while the other datasets were originally sampled at at least 16 kHz. Before training all the audio from Fisher and Switchboard datasets Because of this mixed sample rate training our models can be used to recognize both narrowband (8kHz) and wideband speech (16kHz)."
```

Comment:
```suggestion
    "Among these, Fisher and Switchboard datasets are conversational telephone speech datasets with audio sampled at 8 kHz while the other datasets were originally sampled at at least 16 kHz. Before training, all audio files from Fisher and Switchboard datasets were upsampled to 16 kHz. Because of this mixed sample rate training, our models can be used to recognize both narrowband (8kHz) and wideband speech (16kHz)."
```

Reply:
Done